doc/nrf/migration/migration_guide_1.x_to_2.x.rst

```diff
@@ -0,0 +1,430 @@
+    |                                   | in a single call.                 |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_create_array`        | `zcbor_list_start_encode`         |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_create_map`          | `zcbor_map_start_encode`          |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encoder_close_container`    | `zcbor_map_end_encode`            |
+    |                                   | `zcbor_list_end_encode`           |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_int`                 | `zcbor_int32_put`                 |
+    |                                   | `zcbor_int64_put`                 |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_uint`                | `zcbor_uint32_put`                |
+    |                                   | `zcbor_uint64_put`                |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_enocode_boolean`            | `zcbor_bool_put`                  |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_half_float`          | no equivalent                     |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_float`               | `zcbor_float32_put`               |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_double`              | `zcbor_float64_put`               |
+    |                                   | `zcbor_double_put`                |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_text_string`         | `zcbor_tstr_encode`               |
+    |                                   | `zcbor_tstr_encode_ptr`           |
+    |                                   | `zcbor_tstr_put_lit`              |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_text_stringz`        | `zcbor_tstr_put_term`             |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_byte_string`         | `zcbor_bstr_encode`               |
+    |                                   | `zcbor_bstr_encode_ptr`           |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_null`                | `zcbor_nil_put`                   |
+    +-----------------------------------+-----------------------------------+
+    | `cbor_encode_undefined`           | `zcbor_undefined_put`             |
+    +-----------------------------------+-----------------------------------+
```

Comment:
Since they are function calls, ``function_name()`` could be used throughout the table.

Reply:
done